packages/web-client/app/components/card-pay/create-merchant-workflow/index.ts

```diff
@@ -0,0 +1,147 @@
+import Component from '@glimmer/component';
+import { getOwner } from '@ember/application';
+import { inject as service } from '@ember/service';
+import { WorkflowMessage } from '@cardstack/web-client/models/workflow/workflow-message';
+import { Workflow, cardbot } from '@cardstack/web-client/models/workflow';
+import { Milestone } from '@cardstack/web-client/models/workflow/milestone';
+import { WorkflowCard } from '@cardstack/web-client/models/workflow/workflow-card';
+import PostableCollection from '@cardstack/web-client/models/workflow/postable-collection';
+import NetworkAwareWorkflowCard from '@cardstack/web-client/components/workflow-thread/network-aware-card';
+import NetworkAwareWorkflowMessage from '@cardstack/web-client/components/workflow-thread/network-aware-message';
+import Layer2Network from '@cardstack/web-client/services/layer2-network';
+import { action } from '@ember/object';
+
+import { currentNetworkDisplayInfo as c } from '@cardstack/web-client/utils/web3-strategies/network-display-info';
+
+const FAILURE_REASONS = {
+  DISCONNECTED: 'DISCONNECTED',
+} as const;
+
+class CreateMerchantWorkflow extends Workflow {
+  name = 'Merchant Creation';
+  milestones = [
+    new Milestone({
+      title: `Connect ${c.layer2.fullName} wallet`,
+      postables: [
+        new WorkflowMessage({
+          author: cardbot,
+          message: `Hello, nice to see you!`,
+        }),
+        new WorkflowMessage({
+          author: cardbot,
+          message: `To receive payment through Card Pay, you need to create a merchant account.
+          All you need is to choose a name for your merchant account and sign a transaction to create your merchant on the ${c.layer2.fullName}.`,
+        }),
+        new NetworkAwareWorkflowMessage({
+          author: cardbot,
+          message: `Looks like you've already connected your ${c.layer2.fullName} wallet, which you can see below.
```

Comment:
```suggestion
          message: `Looks like you’ve already connected your ${c.layer2.fullName} wallet, which you can see below.
```

Reply:
agreed, thanks. This came from a copypaste and I see it in a couple of other places, I’ll open another PR for that.